src/citrine/informatics/design_candidate.py

```diff
@@ -0,0 +1,104 @@
+from typing import Type
+from citrine._serialization import properties
+from citrine._serialization.polymorphic_serializable import PolymorphicSerializable
+from citrine._serialization.serializable import Serializable
+
+
+__all__ = [
+    'DesignCandidate',
+    'DesignMaterial',
+    'DesignVariable',
+    'MeanAndStd',
+    'CategoriesAndProbabilities',
+    'Mixture',
+    'ChemicalFormula',
+    'MolecularStructure',
+]
+
+
+class DesignVariable(PolymorphicSerializable["DesignVariable"]):
+    """docstring for DesignVariable"""
+
+    def __init__(self, arg):
+        pass  # pragma: no cover
+
+    @classmethod
+    def get_type(cls, data) -> Type[Serializable]:
+        """Return the subtype."""
+        return {
+            "R": MeanAndStd,
+            "C": CategoriesAndProbabilities,
+            "M": Mixture,
+            "F": ChemicalFormula,
+            "S": MolecularStructure
+        }[data["type"]]
+
+
+class MeanAndStd(Serializable["MeanAndStd"], DesignVariable):
+    """docstring for MeanAndStd"""
+
+    mean = properties.Float('m')
+    std = properties.Float('s')
+
+    def __init__(self):
+        pass  # pragma: no cover
+
+
+class CategoriesAndProbabilities(Serializable["CategoriesAndProbabilities"], DesignVariable):
+    """docstring for MeanAndStd"""
+
+    category_probability = properties.Mapping(properties.String, properties.Float, 'cp')
+
+    def __init__(self):
+        pass  # pragma: no cover
+
+
+class Mixture(Serializable["Mixture"], DesignVariable):
+    """docstring for MeanAndStd"""
+
+    category_probability = properties.Mapping(properties.String, properties.Float, 'cp')
```

Comment:
This looks copy/pasted and needs to be updated.

Reply:
woops, had this fixed locally but didn't commit